src/daemon/daemon.cpp

```diff
@@ -236,6 +236,21 @@ bool Daemon::parseConfig(const QJsonObject& obj, InterfaceConfig& config) {
     }
   }
 
+ {  // Read Split Tunnel Apps
+      QJsonValue value = obj.value("vpnDisabledApps");
```

Comment:
I would like to see the split tunnelling API moved into its own Daemon method so that it can be configured independently without needing to deactivate the VPN (if the platform supports it), which is why it wound up being implemented differently on Linux. Is it possible on Windows for apps to be changed from protected to unprotected while the VPN is connected?

If Linux is the outlier, it would also be reasonable to refactor the Linux implementation to get the app list from the JSON config like we do here.

Reply:
>I would like to see the split tunnelling API moved into its own Daemon method so that it can be configured independently without needing to deactivate the VPN (if the platform supports it), which is why it wound up being implemented differently on Linux. Is it possible on Windows for apps to be changed from protected to unprotected while the VPN is connected?

That would make sense, on windows it's not a problem to update the split-tunnel config as we need. :)